build/yamls/base/conf/antrea-agent.conf

```diff
@@ -120,3 +123,13 @@ featureGates:
 # Provide the address of Kubernetes apiserver, to override any value provided in kubeconfig or InClusterConfig.
 # Defaults to "". It must be a host string, a host:port pair, or a URL to the base of the apiserver.
 #kubeAPIServerOverride: ""
+
+# The virtual IP for NodePort Service support. It must be a link-local IP otherwise the Agents will report error.
```

Comment:
Probably change Agents to antrea-agent to be consistent with other references in this file.

Reply:
Done.